src/NUnitFramework/tests/Internal/EventListenerTextWriterTests.cs

```diff
@@ -79,7 +81,7 @@ public void TestWriteStringArg1()
             ListenerWriter.Write(format, arg0);
             ListenerWriter.WriteLine(format, arg0);
 
-            var expected = "20 Apr 2017";
+            var expected = $"20 {CultureInfo.CurrentCulture.DateTimeFormat.GetAbbreviatedMonthName(4)} 2017";
```

Comment:
`CultureInfo.InvariantCulture` here would still return `"20 Apr 2017"`, and that's not what a `Date` formatted with `"{0:dd MMM yyyy}"` returns on non-English system. In LessThanConstraintTests the `Assert` exception message is formatted using dot decimal mark, so using `CultureInfo.InvariantCulture` ensures that the expected message string contains dot decimal mark formatted double.